openapi/data/openapi_demo.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 Ivan Yelizariev <https://it-projects.info/team/yelizariev>
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html). -->
+<odoo noupdate="1">
+    <record id="namespace_demo" model="openapi.namespace">
+        <field name="name">demo</field>
+        <field name="log_request">debug</field>
+        <field name="log_response">debug</field>
+    </record>
+
+    <record id="ir_exports_res_partner_one_demo" model="ir.exports">
+        <field name="resource">res.partner</field>
+        <field name="export_fields" eval="[
+(0, 0, {'name': 'name'}),
+(0, 0, {'name': 'email'}),
+(0, 0, {'name': 'phone'}),
+(0, 0, {'name': 'website'}),
+(0, 0, {'name': 'function'}),
+(0, 0, {'name': 'company_id'}),
+(0, 0, {'name': 'contact_address'}),
+(0, 0, {'name': 'user_ids'}),
+(0, 0, {'name': 'active'}),
+]"/>
+    </record>
+
+    <record id="ir_exports_res_partner_many_demo" model="ir.exports">
+        <field name="resource">res.partner</field>
+        <field name="export_fields" eval="[
+(0, 0, {'name': 'name'}),
+(0, 0, {'name': 'email'}),
+(0, 0, {'name': 'company_id'}),
+]"/>
+    </record>
+
+    <record id="context_res_partner_demo" model="openapi.access.create.context">
+        <field name="name">default</field>
+        <field name="model_id" ref="base.model_res_partner"/>
+        <field name="context">{'default_function': 'CEO'}</field>
+    </record>
+
+    <record id="access_res_partner_demo" model="openapi.access">
+        <field name="namespace_id" ref="namespace_demo"/>
+        <field name="model_id" ref="base.model_res_partner"/>
+        <field name="api_create" eval="True"/>
+        <field name="api_read" eval="True"/>
+        <field name="api_update" eval="True"/>
+        <field name="api_delete" eval="True"/>
+        <field name="api_public_methods" eval="True"/>
```

Comment:
https://github.com/yelizariev/addons-dev/blob/967fefd040359d489a3a821499496dfb6815c018/openapi/models/openapi_access.py#L35

I think there, it should be "all public methods are ~not~ allowed"

also: https://github.com/yelizariev/addons-dev/blob/967fefd040359d489a3a821499496dfb6815c018/openapi/models/openapi_access.py#L44

Reply:
The correct is:

```
        'When empty -- all public methods are allowed')
         'When empty -- private methods are not allowed')
```

See comments: 
https://github.com/yelizariev/addons-dev/blob/87f5970dc4021110443395d713560c8270b6b5c7/openapi/models/openapi_access.py#L36-L44
https://github.com/yelizariev/addons-dev/blob/87f5970dc4021110443395d713560c8270b6b5c7/openapi/models/openapi_access.py#L22-L35

Reply:
I.e. all private methods could not be allowed, while public methods could